src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from collections import defaultdict
+from datetime import datetime, timezone
+from enum import Enum
+from typing import Optional
+
+from validataclass.dataclasses import validataclass
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator, EnumValidator, IntegerValidator, ListValidator, Noneable, StringValidator
+
+from parkapi_sources.converters.base_converter.pull import GeojsonFeatureGeometryInput
+from parkapi_sources.models import RealtimeParkingSiteInput, StaticParkingSiteInput
+from parkapi_sources.models.enums import ParkAndRideType, ParkingSiteType, PurposeType
+
+
+@validataclass
+class OpenDataSwissAddressInput:
+    addressLine: Optional[str] = Noneable(StringValidator())
+    city: Optional[str] = Noneable(StringValidator())
+    postalCode: Optional[str] = Noneable(StringValidator())
+
+
+class OpenDataSwissCapacityCategoryTypeInput(Enum):
```

Comment:
We usually use `Input` for `validataclasses`, not for Enums. I would suggest to call it `OpenDataSwissCapacityCategoryType`, or even `OpenDataSwissCapacityCategory`, because `Category` is already quite clear about what that is.

Reply:
Thanks for the hint